torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -696,7 +697,7 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
   auto stream = at::cuda::getCurrentCUDAStream();
   executor_utils::initializeCudaContext();
   TORCH_INTERNAL_ASSERT(lowered_);
-  LaunchParams launch_params;
+  launch_params_ = LaunchParams();
```

Comment:
Why is `launch_params` changed to be a class member?

Reply:
Is it because we want to get that information later? For example, in the reduction benchmark file:

```
auto lparams = toString(compile_log.fusion_executor->lastLaunchParams());
```

Reply:
Yes, it's just to hold on to it after the kernel execution so we can fetch it consistently with all the other information. The general approach makes a lot of things stateful that don't need to be, but are in this instance only used to print information.